Create ui-c8y-1020-13-0-user-must-reset-password-on-next-login-is-default-option-5660.md (#2309)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-13-0-user-must-reset-password-on-next-login-is-default-option-5660.md

---
date: ""

> title: The application suggests that a new user set a password during the first login.

Reviewer: ```suggestion
title: "Reset password on next login" option now default if "Password reset link as email" is deselected
```

> title: user must reset password on next login is default option (#5660) [GRAFT][release/cd] (#6777)

Author: The Claude AI suggested to use the following title:
```suggestion
title: "User must reset password on next login" is now the default option when creating a new user
```

> title: Reset password on next login option now default if Password reset link as email is deselected

Reviewer: ```suggestion
title: Reset password on next login option is now the default if Password reset link as email is deselected
```

> title: "Reset password on next login" option now default if "Password reset link as email" is deselected

Reviewer: ```suggestion
title: Reset password on next login option now default if Password reset link as email is deselected
```

product_area: Platform services
change_type:

>   - value: change-QHu1GdukP

Reviewer 2: since the ticket is improvement, shouldn't it be :
value: change-2c7RdTdXo4
label: Improvement

    label: Feature
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58077
version: 1020.13.0
---

> user must reset password on next login is default option (#5660) [GRAFT][release/cd] (#6777)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, when creating a new user in the Cumulocity IoT platform, the "User must reset password on next login" option was not enabled by default. This could lead to potential security risks if the user did not change their password after the first login. With this change, the "User must reset password on next login" option is now enabled by default when creating a new user. This ensures that users are required to change their password immediately after their first login, enhancing the overall security of user accounts in the platform.
```

> Previously, a new user would receive an email with a link to set their password. However, now when "Send password reset link as email" is unchecked, the "User must reset password on next login" option is selected by default. This means the new user must change their password during their first login.

Reviewer: ```suggestion
Previously, a new user received an email with a link to set their password. Now when the checkbox **Send password reset link as email** is cleared, the **User must reset password on next login** option is selected by default. This means the new user must change their password during their first login.
```

---
date: ""

> title: Reset password on next login option now default if Password reset link as email is deselected

Reviewer 3: Ideally it should read:
Reset password on next login option is now the default if the Password reset link as email is deselected.

Reviewer: > Ideally it should read: Reset password on next login option is now the default if the Password reset link as email is deselected.

The title is a heading and not a complete sentence which is why it does not end with a full stop. But we can make the other changes.
